vector3d.h

```diff
@@ -0,0 +1,83 @@
+//3d vector class for Rendera, Yashas M Salian
+
+#include<math.h>
+
+class vec3
+{
+private:
+    float x, y, z;    //Components of vector are kept private
+public:
+    void xinput(float xin)    //functions used to input x, y, z
```

Comment:
These functions don't make much sense. Why not just make a constructor which takes the x, y, z and values we want?